### ISSUE TITLE
Rate limit "found a container with the "k8s" prefix, but too few fields" spam in kubelet logs

### ISSUE THREAD

**Issue by zmerlynn** [GoogleCloudPlatform/kubernetes#5209]
This is super spammy and distracting, on the order of about a screen of text a minute:

```
W0309 21:19:18.193471    4766 docker.go:726] found a container with the "k8s" prefix, but too few fields (5): "k8s_cadvisor.8d424740_cadvisor-agent.file-6bb810db-e2e-test-zml-minion-o8wj.file_7fa8430d20aa229a3c9f14850963a625_52bafe5f"
W0309 21:19:18.193545    4766 docker.go:726] found a container with the "k8s" prefix, but too few fields (5): "k8s_POD.252debe0_cadvisor-agent.file-6bb810db-e2e-test-zml-minion-o8wj.file_7fa8430d20aa229a3c9f14850963a625_5f0e88fd"
```

**Comment by gmarek** [comment]
Yeah. I find that annoying as well. I believe that in the docker.go:722 there should be 5 instead of 6, judging from the comment. I'm sending a PR.

**Comment by vmarmol** [comment]
Re-purposing this bug and will change title, stay tuned...

**Comment by vmarmol** [comment]
Spoke too soon I think :) We should rate limit this log. Sorry to close the PR @gmarek, we should change it to a rate-limit instead.

After #5103 the Kubelet has changed the naming convention of the containers. If there are any old containers this gets logged very often. The fix is to upgrade your containers. It is a bit hard to do with the GCE setup today, but work is in progress to improve that. In the short term we should reduce the amount of logging this causes.

**Comment by vmarmol** [comment]
Closing in favor of #5372
